pandas/tests/frame/methods/test_sort_index.py

```diff
@@ -229,3 +229,84 @@ def test_sort_index_intervalindex(self):
         )
         result = result.columns.levels[1].categories
         tm.assert_index_equal(result, expected)
+
+    @pytest.mark.parametrize(
+        "original_dict, sorted_dict, ascending, ignore_index, output_index",
+        [
+            ({"A": [1, 2, 3]}, {"A": [3, 2, 1]}, False, True, [0, 1, 2]),
+            ({"A": [1, 2, 3]}, {"A": [1, 2, 3]}, True, True, [0, 1, 2]),
+            ({"A": [1, 2, 3]}, {"A": [3, 2, 1]}, False, False, [2, 1, 0]),
+            ({"A": [1, 2, 3]}, {"A": [1, 2, 3]}, True, False, [0, 1, 2]),
```

Comment:
Minor but can you use a non-default index as part of the expectation? Right now the parametrization can't disambiguate between the default index and `ignore_index` argument

Reply:
thanks! changed!